### PR TITLE
Sir Updated Each And Every Thing Accordingly 

### PR DESCRIPTION
**Pull request by Harshith1263**
Sir, I've Changed All The Dependencies To What They Are In the Past !!

### CODE REVIEW

File: Onboarding_App/.dart_tool/version [hunk 1]
```diff
@@ -1 +1 @@
-2.2.3
```
Comment (Safnaj) [inline Onboarding_App/.dart_tool/version:1]: Delete this file under Onboarding_App

**Review by Safnaj — CHANGES_REQUESTED** [review]

File: Onboarding_App/.dart_tool/version (hunk 1 above)
Comment (Harshith1263) [reply Onboarding_App/.dart_tool/version:1]: But Sir In the Original File It Is There Sir !!

**Review by Safnaj — APPROVED** [review]
Thanks for Contributing !!

**Comment by Harshith1263** [comment]
Thank You So Much Sir !!

**Pull request merged**